src/store/modules/data/crawler.js

```diff
@@ -21,33 +20,29 @@ function getInitialState() {
 
 export const crawlerError = createAction(CRAWLER_ERROR)
 const startCrawlerAction = createAction(CRAWLER_START)
-const startCrawlerIPFS = createAction(CRAWLER_START_IPFS)
 export const finishCrawler = createAction(CRAWLER_FINISH)
 
-const crawlerMap = {
-  yelp: getYelpReviews,
-  upwork: getUpWorkReviews
-}
-
-const transformMap = {
-  yelp: transformYelpData,
-  upwork: transformUpworkData
-}
-
 export function startCrawler(type, url) {
   return async (dispatch, getState) => {
     const state = getState()
     dispatch(startCrawlerAction())
     try {
-      const apifyResults = await crawlerMap[type](url)
-      const results = transformMap[type](apifyResults)
-      dispatch(startCrawlerIPFS())
       const signedInDid = get(state, 'data.wallet.did', null)
       const signedOutDid = get(state, 'components.createWallet.walletCreated.did', null)
       const did = signedInDid || signedOutDid
-      const chluIpfs = await getChluIPFS()
-      await chluIpfs.importDID(did)
-      await importUnverifiedReviews(results)
+
+      const result = await fetch(`${API_URL}/api/v1/crawl`, {
+        method: 'POST',
+        body: JSON.stringify({ type, url, did }),
```

Comment:
This is really bad: we are sending the did __with the private key__ to the server. We should only send `did.publicDidDocument.id`